### PR TITLE
Run the test against an existing project; don't start Prometheus.

### LINKED ISSUE #2: Adapt test to start stackdriver-prometheus-sidecar
The main thing is that the namespaces match between the e2e test and the deployment being tested. See the release process at https://github.com/Stackdriver/stackdriver-prometheus-sidecar/blob/master/RELEASE-PROCESS.md

Some easy options:
* Replace the templates for starting Prometheus with the ones in the sidecar repo under kube.
* Add an option for passing the namespace to the e2e test from the command line.

### PR DESCRIPTION
**Pull request by jkohen**
Closes #2

### CODE REVIEW

File: e2e_test.go [hunk 1]
```diff
@@ -14,110 +14,44 @@ limitations under the License.
...
+	if *projectID == "" {
+		t.Fatalf("the project ID must not be empty")
 	}
-	defer func() {
-		if err := execKubectl("delete", "namespace", namespaceName); err != nil {
-			t.Logf("Failed to run kubectl: %v", err)
-		}
-	}()
-	const containerName = "prometheus" // From prometheus-service.yml.
+	t.Logf("Project ID name: %s", *projectID)
```
Comment (StevenYCChou) [inline e2e_test.go]: `"Project ID: %s"` is better.

File: Makefile [hunk 2]
```diff
@@ -11,14 +11,9 @@
...
-START_PROMETHEUS=true
-
 all: test-integration
 
 test-integration:
-ifndef CLUSTER_NAME
-  $(error CLUSTER_NAME is undefined)
-endif
-	go test ./... -args -integration -cluster-name=$(CLUSTER_NAME) -start-prometheus=$(START_PROMETHEUS)
+	go test ./... -args -integration -cluster-location=$(GCP_REGION) -cluster-name=$(KUBE_CLUSTER) -namespace-name=$(KUBE_NAMESPACE) -project-id=$(GCP_PROJECT)
```
Comment (StevenYCChou) [inline Makefile:17]: As we have multiple required parameters to run e2e test, add a section under README on how you run it helps. for example:
```
GCP_REGION=us-east-1b KUBE_CLUSTER=e2e-cluster KUBE_NAMESPACE=test-namespace GCP_PROJECT=test-project make test-integration
```

**Review by StevenYCChou — APPROVED** [review]
Minor comments which can be addressed easily in my opinion, so the PR LGTM. Approve now because we are in different timezone.

File: Makefile (hunk 2 above)
Comment (jkohen) [reply Makefile:17]: Good idea.

**Pull request merged**